<ui version="4.0" >
 <class>surfaceEquationEditor</class>
 <widget class="QWidget" name="surfaceEquationEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>54</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>4</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title" >
      <string>Edit Algebraic Surface Equation</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="surfaceEquation" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>61</height>
         </size>
        </property>
        <property name="text" >
         <string>= 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="surfaceEquationOkButton" >
        <property name="toolTip" >
         <string>Process this surface.</string>
        </property>
        <property name="text" >
         <string>Go!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="helpButton" >
        <property name="toolTip" >
         <string>Display help. (F1)</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="resource.qrc" >:/resources/help.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>surfaceEquation</tabstop>
  <tabstop>surfaceEquationOkButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>surfaceEquation</sender>
   <signal>activated(int)</signal>
   <receiver>surfaceEquationOkButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>174</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel" >
     <x>299</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
